tests/integration/cattletest/core/test_cluster.py

```diff
@@ -0,0 +1,139 @@
+__author__ = 'sonchang'
```

Comment:
In general we don't add author text to source files.  `git blame` is better and more accurate.